test/builtin_dataset_mocks.py

```diff
@@ -62,27 +62,47 @@ def _parse_mock_info(self, mock_info):
 
         return mock_info
 
-    def prepare(self, config):
+    def load(self, config):
         # `datasets.home()` is patched to a temporary directory through the autouse fixture `test_home` in
         # test/test_prototype_builtin_datasets.py
         root = pathlib.Path(datasets.home()) / self.name
-        root.mkdir(exist_ok=True)
+        mock_data_folder = root / "__mock__"
```

Comment:
as we discussed offline, we could also use a more global `home() / cache /` folder where each sub-folder would contain the data for a specfic dataset and config. Looking at our config, look like they should be hashable fairly easily. But we should definitely leave this for another PR.

Reply:
> as we discussed offline, we could also use a more global `home() / cache /` folder

We can, but it can't be within `home()`, since that changes with every call.

Reply:
I assume we can make the `test_home()` fixture global for the test run?